code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -33,9 +33,10 @@
 			if(M.confused < drunk_value)
 				M.confused += 3
 		if(volume >= boozepwr*3.8)
-			M.adjustToxLoss(1)
-	..()
-	return
+			M.adjustToxLoss(1, 0)
+			. = 1
+	return ..() | .
```

Comment:
this is because consumable/on_mob_life() can have effects that needs to call mob updates.